mono/utils/mono-threads-coop.c

```diff
@@ -508,18 +508,16 @@ mono_threads_assert_gc_unsafe_region (void)
 	MONO_REQ_GC_UNSAFE_MODE;
 }
 
-static MonoThreadsSuspendPolicy
+static int
 threads_suspend_policy_default (void)
 {
 #if defined (ENABLE_COOP_SUSPEND)
 	return MONO_THREADS_SUSPEND_FULL_COOP;
-#else
-#if defined (ENABLE_HYBRID_SUSPEND)
+#elif defined (ENABLE_HYBRID_SUSPEND)
 	return MONO_THREADS_SUSPEND_HYBRID;
-#else
+#elif
```

Comment:
this seems like an unintended change. does `#elif` without a condition do anything?

Reply:
oops